.dockerignore

```
@@ -0,0 +1 @@
+.git
```

Add more ignores files here
```suggestion
.git
test/**
spec/**
tmp/**
.gitignore
.hound.yml
.rspec
.rubocop.yml
.travis-yml
docker-compose.yml
README.md
```

---

`749MB` vs `717MB` :tada: